src/openmrs-react-root-decorator.js

```diff
@@ -1,9 +1,17 @@
 import React from "react";
+import { ModuleNameContext } from "@openmrs/esm-module-config";
 
 export default function decorateOptions(opts) {
-  if (typeof opts !== "object" || typeof opts.featureName !== "string") {
+  if (
+    typeof opts !== "object" ||
+    typeof opts.featureName !== "string" ||
+    typeof opts.moduleName !== "string"
+  ) {
     throw new Error(
-      "openmrs-react-root-decorator should be called with an opts object that has a featureName string. e.g. @ErrorBoundary({featureName: 'life'})"
+      "openmrs-react-root-decorator should be called with an opts object that has " +
+        "1. a featureName string and 2. a moduleName string. " +
```

Comment:
Maybe an explanation that `featureName` needs to be **userFriendly** and will be shown to real users. (That's at least the intent of it -- show maybe a modal or something saying "Whoops, the patient chart feature is having problems" to the user.